modules/virt-cloning-vm-snapshot.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-backup-restore-snapshots.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="virt-cloning-vm-snapshot_{context}"]
+= Cloning a VM by using the web console
+
+You can create a new VM by cloning either an existing VM or a VM snapshot.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
```

Comment:
So this procedure explains only the part of cloning an existing VM.
Since you mentioned that `You can create a new VM by cloning either an existing VM or a VM snapshot.` It seems like you also need to explain how to do it from a a vmsnapshot source which is from what I understand is done by going to the snapshot tab and have an action on the snapshot of create vm.

Reply:
I see that it is now explained in modules/virt-creating-vm-from-snapshot-web.adoc
So should be good?